partners_sponsors.md

```diff
@@ -0,0 +1,10 @@
+# Partners and sponsors
+
+Here is a list of Yunohost partners ans sponsors:
+- [GLOBENET](http://www.globenet.org): activist association, at the service of freedom of expression, offering internet services.
+- [TETANEUTRAL-NET](https://tetaneutral.net/): associative Internet access provider currently operating a radio network in Toulouse and its surroundings and a hoster.
+- [GITOYEN](https://gitoyen.net): association bringing together several companies and associations acting as a provider of hosting infrastructure and Internet access.
+- [LDN-NET](https://ldn-fai.net/) : association for the defense of a free, neutral and decentralized Internet whose main means of action is to be an Internet access provider associative and local.
+- [NBS System](https://www.nbs-system.com/): company specialized in hosting, securing Clouds, outsourcing (Information Systems, SaaS Applications, Web Platforms) and managed services.
+- [Framasoft](https://framasoft.org/) : popular education association, a group of friends convinced that an emancipatory digital world is possible, convinced that it will happen thanks to concrete actions on the ground and online with you and for you!
+- [FDN](https://www.fdn.fr/): associative Internet access provider.
```

Comment:
Maybe we could mention the FFDN altogether (and have a special mention / sublist for TTNN, LDN and ARN because they specifically provide help ?)

Reply:
For Framasoft/FDN, I add them without beeing completly sure.... I think that they are more partners than providers. I will make two categories...

Reply:
Let me know if it's better like that

Reply:
It is, thanks ! :+1: :heart: